components/brave_wallet_ui/components/desktop/views/portfolio/components/asset-more-popup/style.ts

```diff
@@ -0,0 +1,76 @@
+// Copyright (c) 2022 The Brave Authors. All rights reserved.
```

Comment:
Can we rename this to `asset-more-popup.style.ts`?

Reply:
Fixed here: f0f6f66ffb7984f3a43025eb001034ee77d394e8